File: 백준/Bronze/2475. 검증수/검증수.cs
using System;

class Q2475
{
    static void Main(string[] args)
    {
        string[] S = Console.ReadLine().Split();
        int A = int.Parse(S[0]);
        int B = int.Parse(S[1]);
        int C = int.Parse(S[2]);
        int D = int.Parse(S[3]);
        int E = int.Parse(S[4]);
        int sum = A * A + B * B + C * C + D * D + E * E;
        int result = sum % 10;
        
        Console.WriteLine(result);
    }
}

File: 백준/Bronze/2475. 검증수/검증수.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int A = sc.nextInt();
        int B = sc.nextInt();
        int C = sc.nextInt();
        int D = sc.nextInt();
        int E = sc.nextInt();
        int sum = A * A + B * B + C * C + D * D + E * E;
        int result = sum % 10;
        
        System.out.println(result);
    }
}